### PR TITLE
Add Noble (Main + Testnet)

### PR DESCRIPTION
**Pull request by boojamya**
This PR...

Is an initial push to get this info up for `noble`. 
Mainnet launch is scheduled for 3/27. 

We will add more peer info post launch.

### CODE REVIEW

File: testnets/nobletestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
```
Comment (JeremyParish69) [inline testnets/nobletestnet/assetlist.json:3]: should be 'nobletestnet'

File: noble/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "ufrienzies",
+      "name": "frienzies",
+      "symbol": "FRNZ"
+    },
+    {
+      "description": "USD Coin",
```
Comment (JeremyParish69) [inline noble/assetlist.json:46]: We have the image for USDC under _non-cosmos/ethereum/usdc.svg, which you may reference from here (or re-upload the image in this folder). And I'd recommend logo images (SVG preferred) for the other assets, too

File: noble/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            "microfrienzies"
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "ufrienzies",
```
Comment (JeremyParish69) [inline noble/assetlist.json:41]: I assume display denom should be 'frenzies' (the unit with 6 exponent

File: noble/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            "microusdc"
+          ]
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "base": "uusdc",
+      "display": "uusdc",
```
Comment (JeremyParish69) [inline noble/assetlist.json:61]: I assume display denom should be 'usdc' (the unit with 6 exponent

File: noble/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+      "name": "usdc",
```
Comment (JeremyParish69) [inline noble/assetlist.json:62]: Can I suggest:
`"name": "USD Coin",`

File: noble/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "ufrienzies",
+      "name": "frienzies",
```
Comment (JeremyParish69) [inline noble/assetlist.json:42]: Can I suggest a capital 'F'?:
`"name": "Frenzies",`

File: testnets/nobletestnet/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+            "microlove"
+          ]
+        },
+        {
+          "denom": "love",
+          "exponent": 6
+        }
+      ],
+      "base": "ulove",
+      "display": "ulove",
```
Comment (JeremyParish69) [inline testnets/nobletestnet/assetlist.json:41]: again, I assume the display denom unit is meant to be 'love' (the one with 6 exponent)

File: noble/chain.json [hunk 8]
```diff
@@ -0,0 +1,62 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "noble",
+  "chain_id": "noble-1",
+  "website": "https://nobleassets.xyz/",
+  "pretty_name": "Noble",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "noble",
```
[the remark this answers is not in the document]
Comment (danbryan) [reply noble/chain.json:9]: https://github.com/satoshilabs/slips/pull/1523

File: testnets/nobletestnet/assetlist.json (hunk 1 above)
Comment (danbryan) [reply testnets/nobletestnet/assetlist.json]: https://github.com/strangelove-ventures/chain-registry/commit/cb3b78bf3125b3bc6842fb09be2bc179b0181caf

File: noble/assetlist.json (hunk 2 above)
Comment (danbryan) [reply noble/assetlist.json:46]: @Jelena647 do you want this to be the `USDC` Logo, the `noble` Logo, or do you have one for `FRNZ`  ?

File: noble/assetlist.json (hunk 3 above)
Comment (danbryan) [reply noble/assetlist.json]: https://github.com/strangelove-ventures/chain-registry/commit/c5fa1f2c8d69515ffec1cde46dc910327e66706f

File: noble/assetlist.json (hunk 6 above)
Comment (danbryan) [reply noble/assetlist.json:42]: https://github.com/strangelove-ventures/chain-registry/commit/c5fa1f2c8d69515ffec1cde46dc910327e66706f

File: testnets/nobletestnet/assetlist.json (hunk 7 above)
Comment (danbryan) [reply testnets/nobletestnet/assetlist.json]: https://github.com/strangelove-ventures/chain-registry/commit/30d279928aa54121917d9f4969582ef961cb8319

File: noble/assetlist.json (hunk 4 above)
Comment (danbryan) [reply noble/assetlist.json]: https://github.com/strangelove-ventures/chain-registry/commit/30ae0d6485a324e1366f6a896520ebf45dad57e4

File: noble/assetlist.json (hunk 5 above)
Comment (danbryan) [reply noble/assetlist.json]: https://github.com/strangelove-ventures/chain-registry/commit/30ae0d6485a324e1366f6a896520ebf45dad57e4

File: noble/assetlist.json (hunk 2 above)
Comment (danbryan) [reply noble/assetlist.json:46]: I'm going to suggest this be approved for now, and will confirm with @Jelena647 if we want to change.

https://github.com/strangelove-ventures/chain-registry/commit/0ce0a29d8bcdf87241f1257839cfc1191a9dc1a0

**Comment by danbryan** [comment]
Thanks @JeremyParish69 I think i got them all.  Please let me know if we missed anything.

File: noble/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+          ]
+        },
+        {
+          "denom": "frienzies",
+          "exponent": 6
+        }
+      ],
+      "base": "ufrienzies",
+      "display": "Frienzies",
+      "name": "frienzies",
```
Comment (JeremyParish69) [inline noble/assetlist.json:42]: Please swap the name and display values here.

File: testnets/nobletestnet/assetlist.json [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+          ]
+        },
+        {
+          "denom": "love",
+          "exponent": 6
+        }
+      ],
+      "base": "ulove",
+      "display": "Love",
+      "name": "love",
```
Comment (JeremyParish69) [inline testnets/nobletestnet/assetlist.json:42]: Please swap the name and display values here.

File: noble/assetlist.json [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+        },
+        {
+          "denom": "usdc",
+          "exponent": 6
+        }
+      ],
+      "base": "uusdc",
+      "display": "usdc",
+      "name": "USD Coin",
+      "symbol": "USD"
```
Comment (JeremyParish69) [inline noble/assetlist.json:63]: Oh, also, I believe the symbol should be 'USDC'

File: noble/assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply noble/assetlist.json:46]: Ok, please place the image into an /images/ folder (within the /noble/ folder) , and then you can refer to that image using
```
"logo_URIs": {
  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/noble/images/frnz.png"
}
```

File: noble/assetlist.json (hunk 11 above)
Comment (boojamya) [reply noble/assetlist.json]: Good catch 🙏

File: testnets/nobletestnet/assetlist.json (hunk 10 above)
Comment (boojamya) [reply testnets/nobletestnet/assetlist.json]: Done, thank you!

File: noble/assetlist.json (hunk 9 above)
Comment (boojamya) [reply noble/assetlist.json]: Done, thank you!

**Comment by Jelena647** [comment]
Hi all, just wanted to clarify that the Noble logo is to identify the Noble chain. Since there is no native token to Noble, we will want to use this logo to identify the chain itself and then update assets issued by Noble with their logo (i.e. FRNZ for the frienzies campaign, USDC for native USDC issuance, etc). Attaching the high-res logo file for Noble here.

Icon
Horizontal_Logo

**Comment by Jelena647** [comment]
Hi all, just wanted to clarify that the Noble logo is to identify the Noble chain. Since there is no native token to Noble, we will want to use this logo to identify the chain itself and then update assets issued by Noble with their logo (i.e. FRNZ for the frienzies campaign, USDC for native USDC issuance, etc). Attaching the high-res logo file for Noble here.

Icon
Horizontal_Logo

**Comment by JeremyParish69** [comment]
> Hi all, just wanted to clarify that the Noble logo is to identify the Noble chain. Since there is no native token to Noble, we will want to use this logo to identify the chain itself and then update assets issued by Noble with their logo (i.e. FRNZ for the frienzies campaign, USDC for native USDC issuance, etc). Attaching the high-res logo file for Noble here.
> 
> Icon Horizontal_Logo

Thanks for providing this. We should add these images to the PR under noble/images/... and refer to them from within the chain.json file using the 'images' property. The PNG image labelled FRNZ is assume is incorrect, and then we just don't have any image for FRNZ right now. (@danbryan )

**Comment by JeremyParish69** [comment]
Also, is there any chance that this 'n' logo would be used for the 'stake' token? Or will that have something different?

**Comment by boojamya** [comment]
> Also, is there any chance that this 'n' logo would be used for the 'stake' token? Or will that have something different?

Yes, the N logo will be for the `stake` token. Still waiting on the Frienzies image. Once I get that, I'll create the image folder and give you a ping. Appreciate it!

**Comment by boojamya** [comment]
@JeremyParish69 should be all set!
SVG images and the usdc image will have to be in a separate PR.

**Pull request merged**